lib/astarte_core/cql_utils.ex

```diff
@@ -167,4 +167,35 @@ defmodule Astarte.Core.CQLUtils do
 
     eid
   end
+
+  @spec realm_name_to_keyspace_name(nonempty_binary(), nonempty_binary()) :: nonempty_binary()
+  def realm_name_to_keyspace_name(realm_name, astarte_instance_id \\ "default")
```

Comment:
The default `astarte_instance_id` should be an empty string, otherwise this would be a breaking change.